### ISSUE TITLE
Issue on Input Shape of network

### ISSUE THREAD

**Issue by taiharry108** [spragunr/deep_q_rl#59]
For the several networks in the build methods, the width and height may be of the wrong order.

**Comment by DanielTakeshi** [comment]
Right, in the code we're supposed to have height before width for our data, but sometimes in `q_network.py` the width is listed first. But since we usually assume the images are square it's not too bad, though it might be nice to have height before width be consistently used for clarity.
